lib/dslash_coarse.in.cu

```diff
@@ -14,6 +14,16 @@ namespace quda {
                                         cvector_ref<const ColorSpinorField> &inB, const GaugeField &Y, const GaugeField &X,
                                         double kappa, int parity, bool dslash, bool clover, const int *commDim, QudaPrecision halo_precision)
   {
+    if (inA.size() > get_max_multi_rhs()) {
```

Comment:
We can't, because the instantiation for the coarse dslash is quite different than that the general instantiation: coarse colors, no double precision, specialization for halo precision etc.